gcc/testsuite/rust/compile/xfail/macro_vis_priv_not_in_follow_set.rs

```diff
@@ -0,0 +1,16 @@
+// { dg-excess-errors "warnings" }
+
+// Check that "priv" is not in the follow set of :vis.
+
+// { dg-error "priv not follow :vis" "#359" { xfail *-*-* } .+1 }
```

Comment:
I believe this should be a `dg-error` instead. `xfail` test category is used for tests that are expected to fail but that should be fixed someday, they highlight a current bug that should be fixed; not tests that are expected to fail in the long term.

Reply:
I moved the test file from `rust/xfail/macro_vis_priv_not_in_follow_set.rs` to `rust/compile/macro48.rs`.
And I also changed the "dg-error" line to:
```Rust
// { dg-error "token `priv` is not allowed after 'vis' fragment" "#359" { target *-*-* } .+2 }
```
Although I am not really sure what every part of that line mean.